examples/nextjs/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  reactStrictMode: true,
+};
```

Comment:
A healthy default for any new project. It was added in #29974 for TypeScript but we forgot about the JS version.